stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListItemStyle.kt

```diff
@@ -0,0 +1,97 @@
+package io.getstream.chat.android.ui.messages.view
+
+import android.content.res.TypedArray
+import androidx.annotation.ColorInt
+import androidx.annotation.StyleableRes
+import java.io.Serializable
+
+internal class MessageListItemStyle private constructor(
+    @ColorInt val messageBackgroundColorMine: Int?,
+    @ColorInt val messageBackgroundColorTheirs: Int?,
+    @ColorInt val messageTextColorMine: Int?,
+    @ColorInt val messageTextColorTheirs: Int?,
+    @ColorInt val messageLinkTextColorMine: Int?,
+    @ColorInt val messageLinkTextColorTheirs: Int?,
+) : Serializable {
+
+    internal companion object {
+        internal const val VALUE_NOT_SET = Integer.MAX_VALUE
+    }
+
+    internal class Builder(private val attributes: TypedArray) {
+        @ColorInt
+        private var messageBackgroundColorMine: Int = 0
```

Comment:
Shouldn't these have `VALUE_NOT_SET` as their default values? Looks like they'd remain `0` if we forget to call one of the builder methods.

Reply:
Oo, good catch. They definitely should.